sql/src/main/java/io/crate/execution/dml/ShardResponse.java

```diff
@@ -257,6 +257,7 @@ public void writeTo(StreamOutput out) throws IOException {
 
         private final BitSet successfulWrites = new BitSet();
         private final BitSet failureLocations = new BitSet();
+        private final List<Object[]> resultValues = new ArrayList<>();
```

Comment:
I am not sure if the `CompressedResult` should have resultValues, it does the job but seems to me semantically questionable.